virt/live_migration/virt-migrating-vm-on-secondary-network.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY 
+[id="virt-migrating-vm-on-secondary-network"]
+= Migrating a virtual machine over a dedicated secondary network
+include::modules/virt-document-attributes.adoc[]
+:context: virt-migrating-vm-on-secondary-network
+
+toc::[]
+
+You can configure a dedicated xref:../../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#attaching-to-multiple-networks[secondary Multus network] for live migration. A dedicated network minimizes disruption to tenant workloads due to network saturation when virtual machine live migration is triggered.
```

Comment:
The second sentence is a little long, and I wonder about the "is triggered" part. Is network saturation a risk only when live migration is first triggered, or is "during live migration" more accurate? (If the former is more accurate, I think it's fine the way it is.)

Reply:
@jean-edouard Regarding Pan's question - Can you please shed some light on which scenario is more accurate?

Reply:
It really goes both ways. If the main network is saturated, migrating over it will be problematic. On the other hand, starting migrations on the main network could take it from an already very busy state to effective saturation... Does that make sense?

Reply:
Thanks, Jed. I think the current wording makes sense for both scenarios. I am inclined to leave the second sentence as is.